apps/financial-aid/backend/src/app/modules/staff/staff.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common'
+import { InjectModel } from '@nestjs/sequelize'
+
+import { StaffModel } from './models'
+
+import { Staff } from '@island.is/financial-aid/shared/lib'
+
+@Injectable()
+export class StaffService {
+  constructor(
+    @InjectModel(StaffModel)
+    private readonly staffModel: typeof StaffModel,
+  ) {}
+
+  findById(id: string): Promise<Staff> {
+    const mockStaff: Staff = {
+      id: id,
+      name: 'Klára Línudóttir',
+      nationalId: '0000000002',
```

Comment:
Not sure if we want to use the user obj here or hard code this, any thoughts?

Reply:
Lets keep it like this for now, good start